logicrunner/logicrunner.go

```diff
@@ -369,6 +373,9 @@ func (lr *LogicRunner) Execute(ctx context.Context, parcel core.Parcel) (core.Re
 		pulse:   lr.pulse(ctx).PulseNumber,
 		result:  make(chan ExecutionQueueResult, 1),
 	}
+	if msg, ok := parcel.Message().(*message.CallMethod); ok && msg.ReturnMode == message.ReturnNoWait {
+		qElement.returnMode = msg.ReturnMode
```

Comment:
maybe message.ReturnNoWait for more clarify

Reply:
why? it will be changed if we implement other methods of return

Reply:
it changes 1 line above anyway, but now you need check above line to understand that msg.ReturnMode is actualy message.ReturnNoWait